src/DependencyInjection/IltarHttpExtension.php

```diff
@@ -29,7 +29,7 @@ public function load(array $config, ContainerBuilder $container)
      */
     private function processRouterConfig(array $config, ContainerBuilder $container, XmlFileLoader $loader)
     {
-        if (false === $config['enabled']) {
+        if (!$this->isConfigEnabled($config)) {
```

Comment:
I see a failed test on this one, the signature is `isConfigEnabled(ContainerBuilder $container, array $config)`